### ISSUE TITLE
Integrate the new welcome screen

### ISSUE THREAD

**Issue by bsclifton** [brave/browser-laptop#7821]
@cezaraugusto has worked with @bradleyrichter to deliver a beautiful new welcome screen (feel free to share details here)

The goal for this issue would be to integrate the screen into Brave. We'll want a new `about:welcome` or similarly named page which loads the content in an iframe.

**Comment by bsclifton** [comment]
@bradleyrichter @cezaraugusto do you all have a link you can share for the welcome screen?

**Comment by cezaraugusto** [comment]
fyi this page will be integrated by using an iframe on about:welcome

**Comment by bsclifton** [comment]
Moving to 0.14.3

**Comment by cezaraugusto** [comment]
after slack convo moving back to 0.15.3

**Comment by bsclifton** [comment]
@cezaraugusto @bradleyrichter this page does exist now, but is there anything triggering it? (ex: first launch?)

**Comment by cezaraugusto** [comment]
only works if you go to about:welcome currently. @bradleyrichter may speak better what is expected. Maybe replace newtab page when first run after update/install happens and back to new tab for the next openings?
